index.bs

```diff
@@ -1693,8 +1687,8 @@ that are returned to the caller when a new credential is created, or a new asser
 
 {{PublicKeyCredential}}'s [=interface object=] inherits {{Credential}}'s implementation of
 {{Credential/[[CollectFromCredentialStore]](origin, options, sameOriginWithAncestors)}}, and defines its own
-implementation of {{PublicKeyCredential/[[Create]](origin, options, sameOriginWithAncestors)}}, {{PublicKeyCredential/[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)}}, and
-{{Credential/[[Store]](credential, sameOriginWithAncestors)}}.
+implementations of {{PublicKeyCredential/[[Create]](origin, options, sameOriginWithAncestors)}}, {{PublicKeyCredential/[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)}}, and
```

Comment:
I think since a user agent only has a single implementation of each of these, the singular might be correct? (Not bothered either way.)

Reply:
Hm, I hadn't thought of it that way. I read this as a group of single implementations each of a single method, but the group being a plural of implementations.

Ah! So I would agree with the singular with this phrasing:

```suggestion
implementation of each of {{PublicKeyCredential/[[Create]](origin, options, sameOriginWithAncestors)}}, {{PublicKeyCredential/[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)}}, and
```